cli_tools_e2e_test/gce_ovf_import/test_suites/ovf_instance_import/ovf_instance_import_test_suite.go

```diff
@@ -46,6 +46,9 @@ var (
 		utils.GcloudProdWrapperLatest:   "gcloud",
 		utils.GcloudLatestWrapperLatest: "gcloud",
 	}
+	// Apply this as instance metadata if the OS config agent is not
+	// supported for the platform or version being imported.
+	skipOSConfig = map[string]string{"osconfig_not_supported": "true"}
```

Comment:
minor: skipOSConfigMetadata, otherwise, the name sounds like a boolean

Reply:
done!